classes/02class/exercises/c02-network02/jaysues/ANSWER.md

```diff
@@ -0,0 +1,26 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18 |10.0.0.4 - 10.0.63.254 |16379 | 10.0.63.255|10.0.0.1,10.0.0.2,10.0.0.3 
```

Comment:
Host range is shown as usable host range, where as actual one will be from 10.0.0.1